app/data/models/stage-execution.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (sequelize, DataTypes) => {
+  const stageExecution = sequelize.define('stage_execution', {
+    id: {
+      type: DataTypes.SMALLINT,
+      primaryKey: true,
+      allowNull: false
+    },
+    applicationReference: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    stageConfigurationId: {
+      type: DataTypes.SMALLINT,
+      allowNull: false
+    },
+    executedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: sequelize.literal('NOW()')
+    },
+    executedBy: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    processedAt: {
+      type: DataTypes.DATE,
+      allowNull: true
+    },
+    action: {
+      type: DataTypes.JSONB,
+      allowNull: true
+    }
+  }, {
+    freezeTableName: true,
+    tableName: 'stage_execution'
+  })
+  stageExecution.associate = function (models) {
+    stageExecution.hasOne(models.application, {
```

Comment:
I think this should be `hasMany` rather than `hasOne` as each application can have multiple execution rows so the same applicationReference can be added more than once.